.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,67 @@
+name: "\U0001F41E Report a problem"
+description: "Report an issue with ESLint or rules bundled with ESLint"
+title: "Bug: (fill in)"
+labels:
+  - bug
+  - "repro:needed"
+body:
+- type: markdown
+  attributes:
+    value: ESLint adheres to the [Open JS Foundation Code of Conduct](https://eslint.org/conduct). By opening an issue, you agree to abide by the code of conduct.
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      Please tell us about how you're running ESLint (Run `npx eslint --env-info`.)
+    value: |
+        Node version: 
+        npm version: 
+        Local ESLint version: 
+        Global ESLint version: 
+        Operating System:
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: What parser are you using?
+    description: |
+      Please keep in mind that some problems are parser-specific.
+    options:
+      - "Default (Espree)"
+      - "@typescript-eslint/parser"
+      - "@babel/eslint-parser"
+      - "vue-eslint-parser"
+      - "@angular-eslint/template-parser"
+      - Other
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What did you do?
+    description: |
+      Please include a *minimal* reproduction case with source code, configuration file, any other information about how you're using ESLint. You can use Markdown in this field.
```

Comment:
The collapsible `<details>` for configuration is pretty handy. There are a [couple](https://github.com/github/feedback/discussions/5271) feedback [discussions](https://github.com/github/feedback/discussions/5250) to have it built-in. Until then, what do you think of providing a `<details>` in the `value` like we do in our current template?

```suggestion
      Please include a *minimal* reproduction case with source code, configuration file, any other information about how you're using ESLint. You can use Markdown in this field.
    value: |
      <details>
      <summary>Configuration</summary>
      <!-- Paste your configuration below: -->
      ```js
      
      ```
      </details>
```

Reply:
Works for me.